machine/v1/types_nutanixprovider.go

```diff
@@ -66,6 +82,20 @@ type NutanixMachineProviderConfig struct {
 	CredentialsSecret *corev1.LocalObjectReference `json:"credentialsSecret"`
 }
 
+// NutanixBootType is an enumeration of different boot types for Nutanix VM.
+type NutanixBootType string
+
+const (
+	// NutanixLegacyBoot is the legacy BIOS boot type
+	NutanixLegacyBoot NutanixBootType = "LEGACY"
+
+	// NutanixUEFIBoot is the UEFI boot type
+	NutanixUEFIBoot NutanixBootType = "UEFI"
+
+	// NutanixSecureBoot is the Secure boot type
+	NutanixSecureBoot NutanixBootType = "SECURE_BOOT"
```

Comment:
These should be PascalCase to match the kube convention, not any upstream constants
```suggestion
	// NutanixLegacyBoot is the legacy BIOS boot type
	NutanixLegacyBoot NutanixBootType = "Legacy"

	// NutanixUEFIBoot is the UEFI boot type
	NutanixUEFIBoot NutanixBootType = "UEFI"

	// NutanixSecureBoot is the Secure boot type
	NutanixSecureBoot NutanixBootType = "SecureBoot"
```

Reply:
Fixed.